charts/oceanbase-cluster/Chart.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: v2
+name: oceanbase-cluster
+description: A Helm chart for Kubernetes to deploy OceanBase cluster
+
+# A chart can be either an 'application' or a 'library' chart.
+#
+# Application charts are a collection of templates that can be packaged into versioned archives
+# to be deployed.
+#
+# Library charts provide useful utilities or functions for the chart developer. They're included as
+# a dependency of application charts to inject those utilities and functions into the rendering
+# pipeline. Library charts do not define any templates and therefore cannot be deployed.
+type: application
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: 4.2.0
```

Comment:
maybe this version is too short, keep this version the same as observer version

Reply:
This field should follow **semver** standard, but our 4.2.0.0-YYYYMMDDHHMMss format is invalid in such style.

Reply:
4.2.0-YYYYMMDDHHMMss is correct, but I think the forth number can not be ignored.

Reply:
or `4.2.0-subpatch.0-101000032023091319`. Major version pattern in semver is "X.Y.Z" which contains three parts, the forth number should be transformed to another form.